modules/nf-core/kraken2/build/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "kraken2_build"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "bioconda::kraken2=2.1.3"
```

Comment:
If module is using a mulled container, where is there only a single entry here?

Reply:
I guess, do you know if there is a list of dependencies for that container?

Reply:
I've asked... I forgot how to do that...

Reply:
```
$ singularity exec ./mulled-v2-1a6fe65bd6674daba65066aa796ed8f5e8b4687b\:688e175eb0db54de17822ba7810cc9e20fa06dd5-0 bash -c 'grep -h requested_spec /usr/local/conda-meta/*.json' | grep -v None | cut -d\" -f4
bwa-mem2=2.2.1
minimap2=2.24
perl=5.32.1
samtools=1.17
staden_io_lib=1.14.14
```